src/coreclr/src/jit/gentree.h

```diff
@@ -3448,6 +3448,56 @@ struct ReturnTypeDesc
     regMaskTP GetABIReturnRegs();
 };
 
+class TailCallSiteInfo
+{
+    bool                   m_isCallvirt : 1;
+    bool                   m_isCalli : 1;
+    CORINFO_SIG_INFO       m_sig;
+    CORINFO_RESOLVED_TOKEN m_token;
+
+public:
+    bool IsCallvirt()
```

Comment:
Please add comments to all these functions. You can see https://github.com/dotnet/runtime/pull/341/files#diff-75141930fdf69093b03d98dc340d1221R3418 for examples.

Reply:
Is the full-blown documentation style really valuable for these simple functions?
It seems like a lot of noise. Would a single-line comment such as `// Is the tailcall a callvirt instruction?` be sufficient?

Reply:
I think single-line comments are appropriate here. https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/clr-jit-coding-conventions.md#9.4 says 
`All functions, except trivial accessors and wrappers, should have a function header comment which describes the behavior and the implementation details of the function.` 
These are trivial accessors so I'll add single-line comments.